lib/ui/home/account/company_profile_creation/profile_creation/company_profile_creation_screen.dart

```diff
@@ -143,7 +144,8 @@ class _CompanyProfileCreationScreenState
                                                     .copyWith(
                                                         company: company)));
                                         SnackBarService.showSnackBar(
-                                            content: 'Successfully!',
+                                            // content: 'Successfully!',
+                                            content: changeSuccessMsgKey.tr(),
```

Comment:
Replacing hard-coded strings with localized keys is recommended for internationalization. However, commenting out the old code instead of removing it can clutter the codebase.

```diff
- // content: 'Successfully!',
+ 
```